docs/source/old/api.md

```diff
@@ -171,7 +172,8 @@ Junction provides API to access information about the conference, schedule, and
 
 - Sample Payload:
 
-```{'text': [{'text': 'Ok', 'id': 1}], 'schedule_item_id': 1, 'choices': [{'id': 1, 'value_id': 1}]}
+```
+{'text': [{'text': 'Ok', 'id': 1}], 'schedule_item_id': 1, 'choices': [{'id': 1, 'value_id': 1}]}
```

Comment:
Would it be better, if we pretty print all of these payload values?

Reply:
Yes, but I didn't want to go about fixing all the old documentation as part of this PR -- for what should be fairly obvious reasons. :)